docs/vast-vpaid.html

```diff
@@ -67,7 +67,6 @@ <h3 class="mb-4">VAST and VPAID</h3>
       playsinline
       controls
       muted
-      autoplay
```

Comment:
Since the non autoplay was causing a bug it seems that autoplay can be a factor, i would provide two examples one with autoplay and one without.